ramls/patron.raml

```diff
@@ -61,6 +61,22 @@ traits:
             required: false
             type: boolean
             default: false
+          offset:
+            description: Skip over a number of elements by specifying an offset value for the query
+            type: integer
+            required: false
+            example: 0
+            default: 0
+            minimum: 0
+            maximum: 2147483647
+          limit:
+            description: Limit the number of elements returned in the response
+            type: integer
+            required: false
+            example: 10
+            default: 0
```

Comment:
My concern is that this is a breaking compatibility change... I'm thinking the default limit value should be `Integer.MAX_VALUE`, so if limit isn't specified by the client we get all records, just like the current implementation.

Reply:
Updated